src/test/create-user.test.ts

```diff
@@ -110,7 +114,59 @@ describe('GraphQL => Unit Test - Create User use-case', () => {
       await createUser.exec(input);
       expect.fail('Should have thrown an error');
     } catch (err: any) {
-      expect(err.message).to.be.equal(`A senha informada não é válida.`);
+      expect(err.message).to.be.eq(`A senha informada não é válida.`);
+    }
+  });
+});
+
+describe('Axios => Unit Testing - Create User use-case', () => {
+  class GraphQLEndPoint {
+    static url: string;
+    static getURL() {
+      const host: string = Container.get(HOST);
+      const port: number = Container.get(PORT);
+      return (this.url = `http://${host}:${port}/`);
     }
+  }
```

Comment:
acho que vc ta complicando demias usando essas classes. Da pra usar direto os valorers das constantes dentro do it e elimitar essa classe

Reply:
Eu posso fazer isso também, claro que fica mais simples... Mas a ideia foi pegar lá do env para testar, então se altera lá não preciso ficar modificando no código